setup.py

```diff
@@ -25,6 +25,7 @@
     'pyramid_duh>=0.1.1',
     'pyramid_jinja2',
     'pyramid_tm',
+    'rsa',
```

Comment:
A quote taken from [this question](http://stackoverflow.com/questions/2573919/creating-signed-urls-for-amazon-cloudfront#answer-16145598) on stack overflow:

boto now supports a dedicated create_signed_url method and the former binary dependency M2Crypto has recently been replaced with a [pure-Python RSA implementation](http://stuvel.eu/rsa) as well, see [Don't use M2Crypto for cloudfront URL signing](https://github.com/boto/boto/pull/1214).

So to give you an answer: yes, we do need the `rsa` dependency to make `create_signed_url` work :)